go/libraries/doltcore/table/editor/table_editor_test.go

```diff
@@ -110,7 +114,7 @@ func TestTableEditorConcurrency(t *testing.T) {
 		}
 		wg.Wait()
 
-		newTable, err := tableEditor.Table()
+		newTable, err := tableEditor.Table(nil)
```

Comment:
Nit: `context.Background()` instead of `nil`. Same goes for the rest of the `Table(nil)`s here.